home2.1/test_java.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Test Case 1: New User
+echo "Test Case 1: New User"
+java task3 Kris
+
+# Test Case 2: Returning User
+echo "Test Case 2: Returning User"
+java task3 Kris
```

Comment:
It is not enough to just run the command - you should also perform expected result verification. And some prerequisite of the test should be done
it looks that you need some helper function for comparison of actual result(AR) and expected result(ER)